pkgs/top-level/all-packages.nix

```diff
@@ -20288,7 +20288,7 @@ in
 
   kgraphviewer = libsForQt5.callPackage ../applications/graphics/kgraphviewer { };
 
-  khal = callPackage ../applications/misc/khal { };
+  khal = python3.pkgs.callPackage ../applications/misc/khal { };
```

Comment:
How can I override this with python38?

Reply:
You are right. Considering this and your other comment, I concluded to take this 1 more step further. The next set of commits follow https://nixos.org/nixpkgs/manual/#python and use `toPythonModule` and use `callPackage` from `python-packages.nix`. I did this for all utilities we package made by `@pimutils`. This includes khard, khal, todoman and vdirsyncer.

As for your particular question, now you should be able to use `python38Packages.khal`. You can do this for all other utilities.